libusb/libusbi.h

```diff
@@ -807,7 +812,8 @@ struct usbi_event_source {
 
 int usbi_add_event_source(struct libusb_context *ctx, usbi_os_handle_t os_handle,
 	short poll_events);
-void usbi_remove_event_source(struct libusb_context *ctx, usbi_os_handle_t os_handle);
+void usbi_remove_event_source(struct libusb_context *ctx, usbi_os_handle_t os_handle)
```

Comment:
This commit lgtm, but it seems you sometimes put the annotation before and sometimes after. Does it indeed accept either way?  Might be nice for consistency to always do it one way or the other.

Reply:
Well, I put it after function declarations, but in front for definitions.

Obviously this worked fine for GCC at least, but I personally don't really care as to the position of these as long as the annotations work.

Reply:
OK I just tried clang and it seems to *require* it in front for definitions, as you did.  And for declarations/prototypes, it accepts before and after.

So this is ok as is.